pydal/migrator.py

```diff
@@ -429,7 +429,7 @@ def migrate_table(self, table, sql_fields, sql_fields_old, sql_fields_aux,
                     drop_expr % (table._rname, key_tmp)
                 ]
                 metadata_change = True
-            elif sql_fields[key]['type'] != sql_fields_old[key]['type']:
```

Comment:
@nextghost ok, but if the issue is that you're migrating on every request because this `if` condition is triggered, then the problem is not the condition itself but the fact that the metadata are different even if they should be identical. I mean: if the problem is that `sql_fields[key]['type'] != sql_fields_old[key]['type']` is `True`, shouldn't we fix the fact that the two types should match instead of dropping the condition? Or am I missing something?